### PR TITLE
Feat/material UI

### PR DESCRIPTION
**Pull request by Alexandra2888**
-I've added Material UI at CSS resources (article & some videos)

### CODE REVIEW

File: components/frontendPageContent/CssFrameworks.js [hunk 1]
```diff
@@ -286,6 +286,49 @@ const CssFrameworks = () => {
...
+
+          <ResourceCardLinks>
+            <ResourceLink>
+              <a target="_blank" href="https://mui.com/">
+                Website
+              </a>
+              <ResourceBadge>Link</ResourceBadge>
+            </ResourceLink>
+            <ResourceLink>
+              <a
```
Comment (Timonwa) [inline components/frontendPageContent/CssFrameworks.js:304]: I cant find a video on this is, it looks more like an article. Can you clarify what this link is about?

File: components/frontendPageContent/CssFrameworks.js [hunk 2]
```diff
@@ -286,6 +286,49 @@ const CssFrameworks = () => {
...
+                Code a Dictionary with React and Material UI ~ FreeCodeCamp
+              </a>
+              <ResourceBadge>Video</ResourceBadge>
+            </ResourceLink>
+            <ResourceLink>
+              <a
+                target="_blank"
+                href="https://www.youtube.com/watch?v=fzxEECHnsvU"
+              >
+                Material UI 5 (MUI) React Tutorial | MUI Responsive Real Project
```
Comment (Timonwa) [inline components/frontendPageContent/CssFrameworks.js:326]: Let this link be the second on the list. It should be directly below the website link. and add the `video` badge to it.

File: components/frontendPageContent/CssFrameworks.js [hunk 3]
```diff
@@ -286,6 +286,49 @@ const CssFrameworks = () => {
...
+                Material UI React ~ codingninjas
+              </a>
+              <ResourceBadge>Video</ResourceBadge>
+            </ResourceLink>
+            <ResourceLink>
+              <a
+                target="_blank"
+                href="https://www.youtube.com/watch?v=ToXna81iij0&t=32s"
+              >
+                Code a Dictionary with React and Material UI ~ FreeCodeCamp
```
Comment (Timonwa) [inline components/frontendPageContent/CssFrameworks.js:317]: Then, let this link be the third on the list.

**Review by Timonwa — CHANGES_REQUESTED** [review]
The resources look good. except for one that confuses me.
Also pls add Material UI to the side menu

File: components/frontendPageContent/CssFrameworks.js (hunk 1 above)
Comment (Alexandra2888) [reply components/frontendPageContent/CssFrameworks.js:304]: Line 297-302: MUI documentation link
```           ResourceLink>
              <ResourceBadge></ResourceBadge>
              <a target="_blank" href="https://mui.com/">
                 Official Website
              </a>
              <ResourceBadge>Link</ResourceBadge>
            </ResourceLink>
            ```

File: components/frontendPageContent/CssFrameworks.js (hunk 2 above)
Comment (Alexandra2888) [reply components/frontendPageContent/CssFrameworks.js:326]: Line 314-322:
```  <ResourceLink>
              <a
                target="_blank"
                href="https://www.youtube.com/watch?v=fzxEECHnsvU"
              >
                Material UI 5 (MUI) React Tutorial | MUI Responsive Real Project
                ~ Lama Dev
              </a>
              <ResourceBadge>Video</ResourceBadge>
            </ResourceLink>
            ```

File: components/frontendPageContent/CssFrameworks.js (hunk 3 above)
Comment (Alexandra2888) [reply components/frontendPageContent/CssFrameworks.js:317]: ✓

**Comment by Alexandra2888** [comment]
Changes done! Hope it's OK now :)
ps: If I have time can I contribute more?

**Review by Timonwa — CHANGES_REQUESTED** [review]
Thank you, @Alexandra2888. Sure, you can contribute as often as you like.

But there is a bug now: an empty badge before the website link. Also, can you remove the CodingNinja link? Its content is confusing.

FireShot Capture 032 - How to Become a Frontend Developer in 2023  - Frontend Developer Road_ - techroadmap-38gj8dfmy-timonwa vercel app

**Comment by Timonwa** [comment]
awesome. 👍🏽

**Review by Timonwa — APPROVED** [review]

**Pull request merged**